50b1d7239aad89000200002d/cmd

```diff
@@ -1 +1 @@
-jM*3Ney%<a-R>a|<esc>I^(<esc>glc).*$<esc>,q
+jM*3Ney%<a-R>a|<esc>I^(<end><backspace>).*$<esc>,q
```

Comment:
Couldn't you apply the same trick to `4fbf8e303be58b0001000024` ? There's a bit that reads `<esc>gld`.